lib/paypal_express/api.rb

```diff
@@ -196,7 +207,21 @@ def build_form_descriptor(kb_account_id, descriptor_fields, properties, context)
         }
         descriptor_fields = merge_properties(descriptor_fields, options)
 
-        super(kb_account_id, descriptor_fields, properties, context)
+        descriptor = super(kb_account_id, descriptor_fields, properties, context)
+
+        # TODO: options need to include return_url and cancel_return_url
+
+        response = @private_api.initiate_express_checkout(options[:account_id],
+                                                          context.tenant_id,
+                                                          options[:amount],
+                                                          options[:currency],
+                                                          options)
+        unless response.success?
+          raise "Unable to initiate paypal express checkout: #{response.message}"
+        end
+
+        descriptor.form_url = @private_api.to_express_checkout_url(response, context.tenant_id, options)
+        descriptor
```

Comment:
On success, can you also add an option to create a `PENDING` payment in Kill Bill? See https://github.com/killbill/killbill-payu-latam-plugin/blob/master/lib/payu_latam/api.rb#L203-L220 for an example (it does it by default in PayU but we probably want the option here to explicitly enable this).

For the option name, let's use `Utils.normalized(options, :create_pending_payment)` to match the option name from [Adyen](https://github.com/killbill/killbill-adyen-plugin/blob/master/src/main/java/org/killbill/billing/plugin/adyen/api/AdyenPaymentPluginApi.java#L124).